lib/App/Config/Chronicle.pm

```diff
@@ -105,9 +105,8 @@ But user is free to implement any storage backend he wants if it is implemented
 =cut
 
 has chronicle_writer => (
-    is       => 'ro',
-    isa      => 'Data::Chronicle::Writer',
-    required => 1,
```

Comment:
What was the reason for removing this?  doesn't seem to fit the description/scope  of the card?

Reply:
ignore that I assumed you where just removing the ping :)